cloud_functions/main.py

```diff
@@ -36,8 +36,13 @@ def clean_and_upload_window(event, context):
     )
 
     window, window_metadata = window_handler.get_window()
-    cleaned_window = window_handler.clean_window(window, window_metadata)
-    window_handler.persist_window(cleaned_window, window_metadata)
+
+    # Correct timestamps with the sensor node time offset
+    for sensor in window["sensor_data"].keys():
```

Comment:
You mean we should send the data from base-station directly into the BigQuerry?

I mean, we can do the operation of updating time-stamps on the rPi, like @thclark  said. 
It was just least intrusive place to put this operation for now, but of course it can be somewhere in the persistence layer.

Reply:
I think that's where it should be eventually but think we should leave it here for now